addons/hr_equipment/views/hr_equipment_views.xml

```diff
@@ -279,26 +279,25 @@
                                 </ul>
                             </div>
                             <div class="oe_kanban_content" tooltip="kanban-tooltip">
-                                <div><b><field name="name"/><span t-if="record.category_id.raw_value">(<field name="category_id"/>)</span></b></div>
```

Comment:
A space is missing between the name and the category (on build 92393), maybe you already fixed it.